cmd/link.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/manifoldco/manifold-cli/api"
+	"github.com/manifoldco/manifold-cli/config"
+	"github.com/manifoldco/manifold-cli/middleware"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	linkCmd := cli.Command{
+		Name:     "link",
```

Comment:
Link could possibly be a future iteration of `init`.

Is there another name we could go with that would convey what's going to happen?

Reply:
Should we make them subcommands instead of using a flag?